docs/getting_started/requirements.md

```diff
@@ -98,7 +95,7 @@ Examples of Experimental setups:
 -   Databases: MSSQL, Oracle (databases technically supported by Doctrine DBAL which we use, but not supported by our installer at the moment, and not covered by automated testing)
 
 Examples of experimental / deprecated bundles:
-- Assetic 2.8 *(As of eZ Platform 2.5LTS, [Webpack Encore](https://symfony.com/doc/5.0/frontend.html) is used for assets. Assetic is no longer actively supported by eZ besides help with migrating code base)*
+- _Currently none, all deprecated system support features have been removed from 3.0_
```

Comment:
```suggestion
```
Let's get rid of the lines at all, if none

Reply:
Well then we should remove the section, right?

Tip: You can mark several lines now to comment on, and then suggest will also work on those lines

Reply:
I also marked the line above to be removed, but GitHub said "nope" to a multi-line comment